mainline.sh

```diff
@@ -31,8 +31,8 @@ shift $((OPTIND-1))
 export CC=clang
 export CXX=clang++
 FUZZ_FLAG="-DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION"
-export CFLAGS="-fsanitize=address"
-export CXXFLAGS="-fsanitize=address -stdlib=libstdc++ $FUZZ_FLAG"
+export CFLAGS="-fsanitize=address,fuzzer-no-link"
```

Comment:
Can you explain this, I'm unsure what it does.

Reply:
As far as I recall, `fuzzer-no-link` adds the instrumentation for fuzzing (like `-fsanitize=fuzzer` would), but without linking libfuzzer in (so you can link it afterwards). When I first tried without this flag, the local mainline.sh builds were not running appropriately, but it might have been a matter of my environment and/or clang version. I can try again and see if it's really needed.